include/labwc.h

```diff
@@ -454,6 +454,20 @@ enum lab_cycle_dir {
 struct view *desktop_cycle_view(struct server *server, struct view *start_view,
 	enum lab_cycle_dir dir);
 
+/**
+ * is_mostly_overlapping - return true if box A or B overlap each other
+ * by more than 80% of their own area
+ */
+int is_mostly_overlapping(struct wlr_box *a, struct wlr_box *b);
```

Comment:
This could be a static function in `desktop.c` as it appears to only be called within that file.

Reply:
Thanks!